modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeEntryLocalServiceImpl.java

```diff
@@ -85,13 +108,22 @@ public ListTypeEntry addListTypeEntry(
 	public ListTypeEntry deleteListTypeEntry(ListTypeEntry listTypeEntry)
 		throws PortalException {
 
-		ListTypeDefinition listTypeDefinition =
-			_listTypeDefinitionPersistence.findByPrimaryKey(
-				listTypeEntry.getListTypeDefinitionId());
+		if (FeatureFlagManagerUtil.isEnabled(
+				listTypeEntry.getCompanyId(), "LPD-24055")) {
 
-		ListTypeDefinitionUtil.validateInvokerBundle(
-			"Only allowed bundles can delete system list type entries",
-			listTypeDefinition.isSystem());
+			ListTypeEntryUtil.validateInvokerBundle(
+				"Only allowed bundles can delete system list type entries",
+				listTypeEntry.isSystem());
+		}
+		else {
```

Comment:
@abesousa I will approve this PR, but can you send a follow-up with a test for this scenario?

`ListTypeEntryLocalServiceTest` -> `testDeleteSystemListTypeEntry`